src/lsrs/helper/tlp-transport.ts

```diff
@@ -0,0 +1,109 @@
+import axios from "axios";
+import {ILogObject, TTransportLogger} from "tslog/src/interfaces";
+
+/**
+ * Domain class representing an error object in the log.
+ */
+class ErrorLog {
+
+    private readonly className: string;
+    private readonly message: string;
+    private readonly stackTrace?: string;
+
+    constructor(error: Error) {
+        this.className = error.constructor.name;
```

Comment:
AFAIK Error instances have a `name` property so you could just use `error.name`.

Reply:
`error.name` gives a different value (I have a couple of unit tests checking this, they expect `GenericError` here, but get `Error`).

Reply:
Yeah, turns out you have to override it manually in your custom classes if I understood the [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/name) right.

Using that could be better if you ever intend to `toString()` the error for whatever reason, but I think this is good as-is.